tox.ini

```
@@ -1,6 +1,11 @@
 [tox]
 envlist = py38,py39,py310,py311,mypy,precom
 
+[pytest]
+filterwarnings =
+    error
+    ignore::DeprecationWarning
```

```suggestion
    default::DeprecationWarning
```

you still want to see them.